source/templates/writing-helpers.md

```diff
@@ -59,11 +59,11 @@ You may also find yourself rendering your view classes in multiple
 places using the `{{view}}` helper. In this case, you can save yourself
 some typing by registering a custom view helper.
 
-For example, let’s say you have a view called `App.CalendarView`.
+For example, let’s say you have a view called `CalendarView`.
```

Comment:
Makes sense. In reality, I would just name it `calendar`, but since it is an example, I thought I should leave it `CalendarView`, but what you are suggesting seems to work better, so I will update accordingly.